### DISCUSSION TITLE
Virtual column: create an expression with a relational value

### DISCUSSION THREAD

**Discussion by GeoGyro** [dbeaver/dbeaver#19112] [Q&A]
I wan't to display a relational value with a virtual column. I defined a foreign key beetween two tables and I want now to see values of a specific field from my joined table, but what is the expression correct syntax ?

Documentation said : `content.xml(columnName, "expression")`

Supposed I have a table `table_a` and `table_b` and foreign key is `table_a.uuid_b`, `table_b.uuid`, I want to display `table_b.name`.

**Comment by mkgrgis** [comment]
There is no valid answer without full context: DB name and version.

**Comment by GeoGyro** [comment]
I'm using a PostgreSQL 13.7 database and DBeaver 22.3.5.202302171528.

**Comment by LonwoLonwo** [comment]
You can use only column names from the current table. You can probably do some view with table_a and table_b columns and add a virtual column.

**Comment by GeoGyro** [comment]
If I understand: a virtual column can't be from a relationnal value ?

**Reply by LonwoLonwo** [reply]
In the expression of the virtual column, you need to use columns from the current table, which contains this virtual column.
